packages/world/ts/config/v2/namespaces.ts

```diff
@@ -19,12 +20,16 @@ export type namespacedTableKeys<world> = world extends { namespaces: infer names
     }[keyof namespaces]
   : never;
 
+export type validateNamespace<namespace, scope extends Scope = AbiTypeScope> = {
```

Comment:
we ought to have the corresponding `validateNamespace` runtime function as well

Reply:
yess that would be neater